app/models/submission.rb

```diff
@@ -143,6 +143,7 @@ def needs_grading_changed?
   before_save :update_if_pending
   before_save :validate_single_submission, :infer_values
   before_save :prep_for_submitting_to_turnitin
+  before_save :prep_for_submitting_to_vericite
```

Comment:
All of the vericite methods that have turnitin variants should be revisited.  Methods that are largely similar except for one or two lines should be rewritten to be generalized.